test/server-handlers.ts

```
@@ -0,0 +1,25 @@
+import { http, HttpResponse } from 'msw'
+
+export const mockUser = jest.fn(
+  (context: { request: Request }): Response =>
+    HttpResponse.json({ firstName: 'John', lastName: 'Smith' }, { status: 200 })
+)
+
+export const mockCsrf = jest.fn(
+  (context: { request: Request }): Response =>
```

ESLint

[error] 'context' is defined but never used.

(@typescript-eslint/no-unused-vars)